### DISCUSSION TITLE
Able to contact npm owner of an package

### DISCUSSION THREAD

**Discussion by jimmywarting** [npm/feedback#836] [Registry]
I would like for there to be a form to send a email to a NPM package owner/maintainer.

- To send a email you also need to be logged in.
- For there to be an option to select to who it should be sent to (just the creator and / or the other maintainers)

**Comment by ljharb** [comment]
Why is that better than manually emailing `npm owner ls $package` or filing a github issue?

**Reply by jimmywarting** [reply]
Oh, didn't know `npm owner ls` existed

can delete this (if you can)
